### DISCUSSION TITLE
Potentially outdated documentation mentioning `model_classes`

### DISCUSSION THREAD

**Discussion by duongdominhchau** [jeremyevans/sequel#2161] [General]
http://sequel.jeremyevans.net/rdoc/files/doc/code_order_rdoc.html

The doc page above mentioned `model_classes`, but I tried `git grep model_classes` on `sequel` repo and only found deprecation notice in the changelog, there is no code for that method. Further searching reveals that the definition is in `sequel_core` version `2.2.0` which is released around 2008. Could someone please tell me what is current best practice about this? Should I just ignore that piece of code and do nothing?

**Comment by jeremyevans** [comment]
`model_classes` isn't a Sequel method.  In the code example, it is placeholder to indicate an array of model classes that you provide.  If you want something Sequel provides, look at the code example directly below that, using the subclasses plugin.  If you think this should be made more obvious, please submit a pull request.

**Reply by duongdominhchau** [reply]
Thank you, I opened a PR for this.
